BlueLevel/TimerRanOutScene.gd

```diff
@@ -0,0 +1,21 @@
+extends CanvasLayer
+
+func _ready():
+	pass
+
+func _process(_delta):
+	pass
+
+func _on_restart_pressed():
```

Comment:
I think the .change_scene_to_file should be called last in this function. I think if it is called first the print and .set_pause will be ignored as the game has changed scenes.